api.js

```diff
@@ -3,7 +3,14 @@ const service = axios.create({
 });
 
 const api = {
-  getAll: () => null, // Return instead a promise containing all the movies,
+  getAll: () => {
+    return new Promise((resolve, reject) => {
```

Comment:
Since axios by default returns a promise, you could simply write here 
`return service.get(/movies).then(response) => { return response.data })`